<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionWin</class>
 <widget class="QDialog" name="QuestionWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;微软雅黑 Light&quot;;
font: 24pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>             写下你的问题
描述精确的问题更易得到解答</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>发布者</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>541</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:5px;
border-top-left-radius:5px;
border:1 solid #BDC0BA;
height:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>标题</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>541</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-right-radius:5px;
border-top-left-radius:5px;
border:1 solid #BDC0BA;
height:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>461</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>内容</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>提问</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>607</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 14pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);
border-radius:8px;
width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
